addons/mrp/models/stock_move.py

```diff
@@ -91,15 +91,6 @@ def _compute_packaging_qtys(self, aggregated_move_lines):
         kit_moves = defaultdict(lambda: self.env['stock.move'])
         kit_qty = {}
 
-        for line in aggregated_move_lines.values():
```

Comment:
Need to check that in further detail, because if we remove that block, then there won't ever be any `kit_aggregated_ml`, `kit_moves` nor `kit_qty`, making that whole method useless.

And while I see a call to `super()`, I don't see the base method anywhere else.